spec/models/conversion_host_spec.rb

```diff
@@ -5,16 +5,17 @@
 
   context "provider independent methods" do
     let(:host) { FactoryGirl.create(:host) }
-    let(:vm) { FactoryGirl.create(:vm_or_template) }
-    let(:conversion_host_1) { FactoryGirl.create(:conversion_host, :resource => host) }
-    let(:conversion_host_2) { FactoryGirl.create(:conversion_host, :resource => vm) }
+    let(:vm) { FactoryGirl.create(:vm) }
+    let(:conversion_host_1) { FactoryGirl.create(:conversion_host, :resource => host, :resource_type => 'Host', :address => '10.0.0.1') }
```

Comment:
If you're setting the `:resource` you shouldn't also set the `:resource_type`, its polymorphic so setting resource will set the id and type automatically.

Reply:
Ok, I didn't think it would validate properly without it, so I've updated it.